### PR TITLE
Add new device weitai equil

### PR DESCRIPTION
**Pull request by pauloved**
add new device
name: Equil Insulin Patch Pump/Equil Insulin Micro Pump
deviceId:MTM-I
intro:It is small and lightweight also the tubeless micro insulin pump has 4 years warranty which could provide 72h insulin supply.
device-pic:
image

### CODE REVIEW

**Comment by pauloved** [comment]
@gniezen  hi?  Why hasn't AppVeyor been passed, i m not understant

File: app/reducers/devices.js [hunk 1]
```diff
@@ -167,6 +168,20 @@ const devices = {
...
+  weitai: {
+    instructions: 'Connect your Equil to your computer',
+    name: 'Equil',
+    image: {
+      src: weitaipump,
+      height: 200,
+      width: 200,
+      alt: 'Weitai link',
+    },
+    key: 'Weitai',
```
Comment (gniezen) [inline app/reducers/devices.js:180]:
```suggestion
    key: 'weitai',
```

Comment (gniezen) [reply app/reducers/devices.js:180]: Also, add

```js
Weitai: usbDevice,
```
to `device.DeviceComms` object (line 113 in `lib/core/device.js`)

**Comment by gniezen** [comment]
Hi @pauloved ! You're welcome to leave this pull request open until you're ready for it to be reviewed.Just mark it as a draft pull request: https://github.blog/2019-02-14-introducing-draft-pull-requests/

And please let me know if you have any questions - it's very exciting to see someone else working on a driver for Uploader! Does this device use the IEEE 11073:10419 USB insulin pump standard?

**Comment by gniezen** [comment]
Ah, I see this PR has now been superseded by #1277 .

**Pull request closed without merging**